eng/common/scripts/Prepare-Release.ps1

```diff
@@ -109,11 +109,15 @@ else
 $releaseDateString = $ParsedReleaseDate.ToString("MM/dd/yyyy")
 $month = $ParsedReleaseDate.ToString("MMMM")
 
-Write-Host
 Write-Host "Assuming release is in $month with release date $releaseDateString" -ForegroundColor Green
+if (Test-Path "Function:GetExistingPackageVersions")
```

Comment:
Instead of querying the package manager we should consider using the tags? If so we can move some of the tag helpers from https://github.com/Azure/azure-sdk/blob/main/eng/scripts/PackageVersion-Helpers.ps1#L3 into the tools repo.

Reply:
Isn't it possible for tags to exist for packages that eventually failed to publish to the package manager?